content/reference/redactors/overview.md

```diff
@@ -0,0 +1,73 @@
+---
+date: 2019-10-23
+linktitle: "Redactors Overview"
+title: "Redactors Overview"
+weight: 20010
+---
+
+Redactors are YAML specifications that define which data to remove when generating a support bundle.
+
+Redactors can be specified as a standalone document, as a global entry in a collector spec, or individually on a single collector.
+
+```yaml
+apiVersion: troubleshoot.replicated.com/v1beta1
+kind: Redactor
+metadata:
+  name: my-redactor-name
+spec:
+  redacts: []
+```
+
+```yaml
+apiVersion: troubleshoot.replicated.com/v1beta1
+kind: Collector
+metadata:
+  name: my-collector-name
+spec:
+  collectors:
+  - data:
+      redactors: []
```

Comment:
I'd like to remove this from the collector element. This isn't in-scope and needs more thoughtful design before implementation. The feature built was for last-mile redactors, specified by the customer, but these docs create a lot of unnecessary confusion for me when I think about why and how a developer would implement them. There are a lot of options, all of which are probably useful, but it's increasing the complexity of understanding the product too much.

Reply:
Can do